lms/djangoapps/discussion/views.py

```diff
@@ -379,7 +379,7 @@ def single_thread(request, course_key, discussion_id, thread_id):
             'threads': threads,
             'roles': utils.get_role_ids(course_key),
             'is_moderator': is_moderator,
-            'thread_pages': query_params['num_pages'],
+            'thread_pages': 1,
```

Comment:
This doesn't feel right.  What happens if this is just removed altogether?  It seems odd to possibly provide the wrong number of pages.  This doesn't seem to be the current page.  Does this cause issues for threads with multiple pages?